api/src/wfl/module/xx.clj

```diff
@@ -27,43 +27,33 @@
 (def workflow-wdl
   "The top-level WDL file and its version."
   {:release "ExternalExomeReprocessing_v2.1.1"
-   :top     "pipelines/broad/reprocessing/external/exome/ExternalExomeReprocessing.wdl"})
+   :top     (str "pipelines/broad/reprocessing/external/exome/"
+              "ExternalExomeReprocessing.wdl")})
 
 (def ^:private references-defaults
-  (let [hg38 (partial str "gs://gcp-public-data--broad-references/hg38/v0/")]
-    {:calling_interval_list      (hg38 "exome_calling_regions.v1.interval_list")
-     :contamination_sites_bed    (hg38 "contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.bed")
-     :contamination_sites_mu     (hg38 "contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.mu")
-     :contamination_sites_ud     (hg38 "contamination-resources/1000g/1000g.phase3.100k.b38.vcf.gz.dat.UD")
-     :dbsnp_vcf                  (hg38 "Homo_sapiens_assembly38.dbsnp138.vcf")
-     :dbsnp_vcf_index            (hg38 "Homo_sapiens_assembly38.dbsnp138.vcf.idx")
-     :evaluation_interval_list   (hg38 "exome_evaluation_regions.v1.interval_list")
-     :haplotype_database_file    (hg38 "Homo_sapiens_assembly38.haplotype_database.txt")
-     :known_indels_sites_vcfs    [(hg38 "Mills_and_1000G_gold_standard.indels.hg38.vcf.gz")
-                                  (hg38 "Homo_sapiens_assembly38.known_indels.vcf.gz")]
-     :known_indels_sites_indices [(hg38 "Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi")
-                                  (hg38 "Homo_sapiens_assembly38.known_indels.vcf.gz.tbi")]
-     :reference_fasta            (references/reference_fasta)}))
-
-(def ^:private scatter_settings-defaults
-  {:haplotype_scatter_count     50
-   :break_bands_at_multiples_of 0})
-
-(def ^:private papi_settings-defaults
-  {:agg_preemptible_tries 3
-   :preemptible_tries     3})
+  (let [hg38 "gs://gcp-public-data--broad-references/hg38/v0/"
+        hsa  "Homo_sapiens_assembly38"]
+    (merge references/hg38-exome-references
+      references/contamination-sites
+      references/exome-scatter-settings
+      {:calling_interval_list (str hg38 "exome_calling_regions.v1.interval_list")
+       :haplotype_database_file (str hg38 hsa ".haplotype_database.txt")})))
 
 (def ^:private workflow-defaults
-  (let [hg38 (partial str "gs://gcp-public-data--broad-references/hg38/v0/")]
+  (let [hg38          "gs://gcp-public-data--broad-references/hg38/v0/"
+        hsa           "Homo_sapiens_assembly38"
+        bait_set_name "whole_exome_illumina_coding_v1"
+        iv1           (str "HybSelOligos/" bait_set_name "/" "." hsa)]
```

Comment:
I really don't like this - I wrote it this way to be explicit. I can look at the design doc and automatically recognise the defaults without juggling temporaries. Please revert.

Reply:
I don't know what you mean about juggling temporaries.
I agree with you about the scatter-settings.
They are back where they belong.

Reply:
FWIW: Wanted to emphasize that inputs depend on the chosen bait set.